cmd/client-s3.go

```diff
@@ -286,6 +286,9 @@ func (c *s3Client) AddNotificationConfig(arn string, events []string, prefix, su
 
 	// Set the new bucket configuration
 	if err := c.api.SetBucketNotification(bucket, mb); err != nil {
+		if ignoreExisting && err.Error() == "An object key name filtering rule defined with overlapping prefixes, overlapping suffixes, or overlapping combinations of prefixes and suffixes for the same event types." {
```

Comment:
You might want to remove the dot in the error message and uses strings.Contains() instead

Reply:
done. @vadmeste  PTAL